Jenkinsfile

```diff
@@ -10,11 +10,18 @@ def testEnvFolder
 
 def testbeds
 
+def cleanAnsiColors(String sourceFile, String outputFile) {
+    def stripAnsiConceal = /sed -e 's|\x1b\[8[^m]*m[^\x1b]*\x1b\[[^m]*m||g'/
+    def stripEscapeCode  = /sed -e 's|\x1b\[[^m]*m||g'/
```

Comment:
MK: could you add comments showing examples of how these regexes work? Like example strings that they match